community_server/src/Controller/AppRequestsController.php

```diff
@@ -352,5 +355,77 @@ private function acquireAccessToken($session_id)
       
     }
     
+    
+    
+    // ********************* klicktipp API ************************************
+
+    public function klicktippSubscribe($session_id = 0)
+    {
+        $login_result = $this->requestLogin($session_id, false);
+        if($login_result !== true) {
+            return $this->returnJson($login_result);
+        }
+        $session = $this->getRequest()->getSession();
+        $user = $session->read('StateUser');
+        
+        $api_key =  Configure::read('KlickTipp_API_KEY');
+        $email_address = $user['email']; // Replace with the email address.
+        $fields = array ( // Use field_index to get all custom fields.
+          'fieldFirstName' => $user['first_name'],
+          'fieldLastName' => $user['last_name'],
+        );
+        
+
+        $connector = new KlicktippConnector();
```

Comment:
Where is the `KlicktippConnector` coming from?

Reply:
It is the Klicktipp PHP-Wrapper